src/main/java/StringCalculator.java

```diff
@@ -0,0 +1,58 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    private final String DEFAULT_PATTERN = "[,:]";
+    private final Pattern PATTERN = Pattern.compile("//(.)\n(.*)");
+    private final String EMPTY_STRING = "";
+    private final int ZERO = 0;
+
+    public long addStringNumber(String text) {
+        if(checkIfNullOrEmpty(text)) return ZERO;
+
+        String delimiter = findDelimiter(text);
+        String[] tokens = text.split(delimiter);
+
+        long sum = 0;
+        for(String token : tokens) {
+            if(checkIfIllegalStringNumber(token) == true) continue;
```

Comment:
아래 조건절에 작성한것 처럼 반환이 `boolean`형이니까 `== true` 없이도 확인이 가능할거 같습니다~
또한 컨벤션 적용이 일부는 되어있고 일부는 안되어있는 것 같은데요~ `ctrl + alt + L` 또는 `Command + Option + L` 단축키를 이용해서 적용해줄 수 있습니다~
```suggestion
if (checkIfIllegalStringNumber(token) {
    continue;
}
```